pepper-apis/studybuilder-cli/src/main/java/org/broadinstitute/ddp/studybuilder/task/RevisionStudyActivityVariablesSupport.java

```diff
@@ -0,0 +1,165 @@
+package org.broadinstitute.ddp.studybuilder.task;
```

Comment:
Generic class which can be extended by other activity version classes. 
Currently supports only versioning existing template variables